### DISCUSSION TITLE
Create two species library in DIA-NN

### DISCUSSION THREAD

**Discussion by Saleh-AU** [vdemichev/DiaNN#1075] [Q&A]
Hi,
I want to create a spectral library from 2 different species? How can I do that in DIA-NN? 
Thank you.

**Comment by vdemichev** [comment]
Hi Saleh,

Just add the two FASTA databases and generate an in silico predicted library thus featuring both species. In all stages of processing, including the generation of this library and the analysis using it, makes sense to set protein inference to Genes (species-specific).

Best,
Vadim

**Reply by Saleh-AU** [reply]
Thank you Dr. Vdemichev.
